### DISCUSSION TITLE
ESP NOW number of nodes

### DISCUSSION THREAD

**Discussion by orbitcoms** [Mair/esp32-course#93] [Course Qeustions] [answered]
Hi Mair,

Regarding the introduction to ESP NOW.

Just to confirm, the 20 node limit is only for paired nodes and you can have any number of networks as long as any network does not exceed 20?

**Comment by Mair** [comment, marked as answer]
Hi @orbitcoms 
Yes correct.
If you want to connect more nodes you can use wifi mesh which will connect hundreds of nodes or BLE mesh which can connect thousands of nodes

**Comment by gmag11** [comment]
https://github.com/Mair/esp32-course/discussions/94#discussioncomment-2859668
